resilience4j-kotlin/build.gradle

```diff
@@ -19,12 +19,22 @@ dependencies {
     testImplementation(project(':resilience4j-timelimiter'))
 }
 
+def experimentalFlags = [
+    "-Xuse-experimental=kotlin.Experimental"
+]
```

Comment:
The experimental flag is only needed for the usage of  https://github.com/resilience4j/resilience4j/blob/05d45eaad00ccd1088e7d30b4e84ca997f233ac5/resilience4j-kotlin/src/main/kotlin/io/github/resilience4j/kotlin/circuitbreaker/FlowCircuitBreaker.kt#L51

The `UseExperiemental` annotation allows the usage of the `onCompletion()` flow operator.  This lets us guarantee the method contract to library consumers without requiring them to also annotate usages of the circuit breaker operator.